### ISSUE TITLE
Where is the model.py

### ISSUE THREAD

**Issue by dongfeng951** [CatherineWong/cs332_final_project#2]
Hi,
    Thank you for the great work. I copy the model.py file from Seq2Seq-PyTorch, but when I run the adversarial.py, there is one bug. The reason is that when executing Line 164, `src_encoding = self.autoencoder_model.encode(input_lines_src.detach())`， there is no such function in Seq2SeqAutoencoder in the model.py. I guess the model.py is not exactly the same as the one in Seq2Seq-PyTorch, can you tell me how to find the correct one?
     Thanks!
